inc/features/blocks/personalization-variant/register.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Personalization Variant Block Server Side.
+ *
+ * @phpcs:ignoreFile HM.Files.FunctionFileName.WrongFile
```

Comment:
This will ignore **all rules** in the file, needs to be `disable` instead:

```suggestion
 * @phpcs:disable HM.Files.FunctionFileName.WrongFile
```